app/ModelSerializers/ExtraQuestionTypeSerializer.php

```diff
@@ -43,6 +43,14 @@ public static function testRule($e){
         return $e->allowsValues();
     }
 
+    public static function shouldSkip($e, $params){
+        if (array_key_exists('attendee', $params)) {
+            $attendee = $params['attendee'];
+            return !$attendee->isAllowedQuestion($e);
```

Comment:
$e is a QuestionRule, u need to get the question instance

Reply:
done